features/dd-sdk-android-trace/src/test/kotlin/com/datadog/opentracing/PendingTraceTest.kt

```diff
@@ -89,7 +272,6 @@ class PendingTraceTest {
                         "childSpan_$i",
                         internalLogger
                     )
-                    pendingTrace.registerSpan(span)
```

Comment:
what is the need for this change?

Reply:
the `registerSpan` it is already called in the constructor of `DdSpan`